opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_12_6/syncCohortsAndSamplesMigration.java

```diff
@@ -0,0 +1,73 @@
+package org.opencb.opencga.app.migrations.v2_12_6;
+
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Projections;
+import com.mongodb.client.model.Updates;
+import org.apache.commons.collections4.CollectionUtils;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import org.opencb.opencga.catalog.db.api.CohortDBAdaptor;
+import org.opencb.opencga.catalog.db.api.SampleDBAdaptor;
+import org.opencb.opencga.catalog.db.mongodb.MongoDBAdaptor;
+import org.opencb.opencga.catalog.db.mongodb.MongoDBAdaptorFactory;
+import org.opencb.opencga.catalog.migration.Migration;
+import org.opencb.opencga.catalog.migration.MigrationTool;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Migration(id = "syncCohortsAndSamplesMigration" ,
+        description = "Sync array of samples from cohort with array of cohortIds from Sample",
+        version = "2.12.6",
+        domain = Migration.MigrationDomain.CATALOG,
+        language = Migration.MigrationLanguage.JAVA,
+        date = 20240621
+)
+public class syncCohortsAndSamplesMigration extends MigrationTool {
```

Comment:
Why lower case? didn't the checkstyle complain on this? :o

Reply:
You can definitely complain. I just changed it :P